### PR TITLE
Enable GitHub Actions builds

### PR DESCRIPTION
**Pull request by turikhay**

### CODE REVIEW

**Comment by xyzeva** [comment]
Hi!

It may be a good idea to get a release done here, thoughts?

**Comment by turikhay** [comment]
I prefer releasing by pushing tags like `v1.0.1` like [here](https://github.com/turikhay/MapModCompanion/blob/main/.github/workflows/github.yml), but this can also be done manually by calling a workflow. What do you think is best?

**Comment by xyzeva** [comment]
Yeah! I meant like pushing tags.

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default development version
+# See: ./.github/workflows/build.yml
+version=0.0-SNAPSHOT
```
Comment (xyzeva) [inline gradle.properties]: Can be changed to `1.0.0`

**Review by xyzeva — APPROVED** [review]
LGTM

File: gradle.properties (hunk 1 above)
Comment (turikhay) [reply gradle.properties]: `1.0.0`, not `1.0` (as it is now)?

Comment (xyzeva) [reply gradle.properties]: `1.0` is fine, probably better.

**Comment by turikhay** [comment]
Releases created by pushing `v*.*` tags [should look like this](https://github.com/turikhay/utilsdetector/releases/tag/v1.0-test).

**Pull request merged**